en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-multiple-gateways-with-choreo-connect.md

```diff
@@ -39,11 +54,35 @@ Let's assume you have two environments called "choreo-connect-1" and "choreo-con
         reconnectInterval = 5000
         reconnectRetryCount = 60
     ```
-        
-2. Start Choreo Connect deployment from docker-compose. 
 
-3. Add the Gateway environments "choreo-connect-1" and "choreo-connect-2" from the Gateways in the APIM admin portal.
+2. Configure gateway environment label in Enforcer and Router. Update the following environment labels in each component. Update the following environment variables defined in the docker-compose file.
```

Comment:
```suggestion
2. Configure the Gateway environment label in the Enforcer and Router. Update the following environment labels in each component. Update the following environment variables defined in the docker-compose file.
```

Reply:
There are too many steps lumped in together here. Can we break this into multiple steps or reword this to make sure people know what they are doing here?

Reply:
```suggestion
2. Next let's configure the Gateway environment label in the Enforcer and Router. For that update the following environment variables in each service in the docker-compose file.
```
Yes we can keep this as a one step, reworded @samgnaniah could you please check this again?